dll/win32/winmm/playsound.c

```diff
@@ -39,6 +40,61 @@ typedef struct tagWINE_PLAYSOUND
 static WINE_PLAYSOUND *PlaySoundCurrent;
 static BOOL bPlaySoundStop;
 
+/* An impersonation-aware equivalent of ExpandEnvironmentStringsW */
+static DWORD PlaySound_ExpandEnvironmentStrings(LPCWSTR lpSrc, LPWSTR lpDst, DWORD nSize)
```

Comment:
Why does this function need PlaySound in its name, is it not a generic expand function? `ExpandImpersonatedEnvironmentStrings()`?

Reply:
It's an internal implementation (private) and specific to `PlaySound` so I named it accordingly.